RoddyCore/src/de/dkfz/roddy/config/ToolFileGroupParameter.groovy

```diff
@@ -46,11 +44,7 @@ class ToolFileGroupParameter extends ToolEntry.ToolParameter<ToolFileGroupParame
 
     @Override
     ToolFileGroupParameter clone() {
-        List<ToolFileParameter> _files = new LinkedList<ToolFileParameter>()
-        for (ToolFileParameter tf : files) {
-            _files.add(tf.clone())
-        }
-        return new ToolFileGroupParameter(groupClass, genericFileClass, _files, scriptParameterName, passOptions)
+        return new ToolFileGroupParameter(groupClass, genericFileClass, scriptParameterName, passOptions)
```

Comment:
O.k. you remove the deep(er) copy.  I assume that the 'files' are not modified or used afterwards anyway anymore, right?

P.S.: This is an excellent example where the reasoning is much easier if value-classes (constant) are being used.